.github/workflows/issue-labels.yml

```diff
@@ -0,0 +1,33 @@
+name: Issue labels
+
+on:
+  issues:
+    types: [opened, reopened]
+
+env:
+  TRIAGERS: '["karthiknadig","eleanorjboyd","anthonykim1"]'
```

Comment:
Is anthony meant to be here? I think he's pivoted to focus on terminal? @karthiknadig